common/app/renderers/DotcomRenderingService.scala

```diff
@@ -417,6 +418,14 @@ class DotcomRenderingService extends GuLogging with ResultWithPreconnectPreload
     val json = DotcomRenderingDataModel.toJson(dataModel)
     post(ws, json, Configuration.rendering.articleBaseURL + "/Article", CacheTime.Facia)
   }
+
+  def getEditionsCrossword(
+      ws: WSClient,
+      crosswords: EditionsCrosswordRenderingDataModel,
+  )(implicit request: RequestHeader): Future[Result] = {
+    val json = EditionsCrosswordRenderingDataModel.toJson(crosswords)
+    post(ws, json, Configuration.rendering.articleBaseURL + "/EditionsCrossword", CacheTime.Facia)
```

Comment:
Why `CacheTime.Facia`?

Reply:
I think generally post requests for articles are using a different cache time based on their metadata:

https://github.com/guardian/frontend/blob/c7b403b8e08ef83e69698cda72131d7407e360cc/common/app/renderers/DotcomRenderingService.scala#L215

https://github.com/guardian/frontend/blob/c7b403b8e08ef83e69698cda72131d7407e360cc/common/app/renderers/DotcomRenderingService.scala#L311

with `CacheTime.Default` as the fallback:

https://github.com/guardian/frontend/blob/c7b403b8e08ef83e69698cda72131d7407e360cc/common/app/model/meta.scala#L326

?

Post requests for fronts and tag pages are using `CacheTime.Facia`, but that's likely correct because they're examples of "facia" content. I agree that probably we should review `getMedia`, `getImageContent` and `getGallery`'s usage of `CacheTime.Facia`; have captured this in #27628.

Given this, I don't think it's what we want to use here. Perhaps `CacheTime.Default` is the correct choice if we don't have a specific, different cache setting for these pages? @SiAdcock @arelra any thoughts?